app/services/cycle_timetable_query.rb

```diff
@@ -1,26 +1,5 @@
-class EndOfCycleTimetable
-  CURRENT_YEAR_FOR_SCHEDULE = 2021
-
-  # These dates are configuration for when the previous cycle ends and the next cycle starts
-  # The 2020 dates are made up so we can generate sensible test data
-  CYCLE_DATES = {
-    2020 => {
-      apply_1_deadline: Date.new(2019, 8, 24),
-      stop_applications_to_unavailable_course_options: Date.new(2019, 9, 7),
-      apply_2_deadline: Date.new(2019, 9, 18),
-      find_closes: Date.new(2019, 10, 3),
-      find_reopens: Date.new(2019, 10, 6),
-      apply_reopens: Date.new(2019, 10, 13),
-    },
-    2021 => {
-      apply_1_deadline: Date.new(2020, 8, 24),
-      stop_applications_to_unavailable_course_options: Date.new(2020, 9, 7),
-      apply_2_deadline: Date.new(2020, 9, 18),
-      find_closes: Date.new(2020, 10, 3),
-      find_reopens: Date.new(2020, 10, 6),
-      apply_reopens: Date.new(2020, 10, 13),
-    },
-  }.freeze
+class CycleTimetableQuery
```

Comment:
Does this renaming make sense?

Reply:
Good question. Is this a fair way to describe them?

- `RecruitmentCycle` represents the whole cycle and operates at the level of _years_
- `CycleTimetableQuery` represents our current position within the cycle and operates at the level of _phases_ within years

For now I think having both is useful — eg we like the _year_ resolution for dashboards, years we show in manage, etc. I wonder if `CycleTimetableQuery` ought to be where `.current_cycle` lives, though? And that could return a `RecruitmentCycle`.

I'm not completely sold on the name `CycleTimetableQuery`. That to me implies that there's one query, and we'd run `call` on the object to get the data — whatever that is — back. Sooner call it something concrete like  `RecruitmentCycleCalendar`?